types/cometd/AckExtension/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as m from '..';
```

Comment:
What does m stand for? Main?

Reply:
"module", I used this template as a base: [module-plugin.d.ts](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html)